tests/unit/test_convert.py

```diff
@@ -118,15 +91,15 @@ def test_copy_attrs(self):
                 dataset.setncattr('attr_three', 'val_three')
                 dataset.setncattr('attr_two', 'val_four')
```

Comment:
```suggestion
                dataset.setncattr('attr_two', 'val_four_wont_be_copied')
```
Totally missed that this was a different attribute value and wouldn't be updated.

Reply:
Good call - that makes it much more transparent what to expect.